pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "truss"
-version = "0.9.16"
+version = "0.9.17"
```

Comment:
This isn't a significant enough change and we haven't tested it end to end yet. May be set this to rc1 so `0.9.17rc1`.